upgrade/steps-helpers.go

```diff
@@ -82,7 +82,7 @@ func originalGoVersionOf(ctx context.Context, repo ProviderRepo, file, needleMod
 	return module.Version{}, false, nil
 }
 
-func prBody(ctx Context, repo ProviderRepo, upgradeTargets UpstreamVersions, goMod *GoMod, targetBridge string) string {
+func prBody(ctx Context, repo ProviderRepo, upgradeTarget *UpstreamUpgradeTarget, goMod *GoMod, targetBridge string) string {
```

Comment:
Will `updateTarget` ever be `nil`? If not, this shouldn't be a pointer. If so, then line 109 will panic at
```go
	if len(upgradeTarget.GHIssues) > 0 {
```
If this needs to be a pointer for dataflow reasons, can we add an `contract.Assertf(upgradeTarget != nil`, "upgradeTarget should always be non-nil")`?